src/lenses/RougeScoreLens.tsx

```diff
@@ -0,0 +1,26 @@
+import { Lens } from "../types";
+import rouge from 'rouge';
+
+const calculateRougeScore=(values: unknown[])=>{
+        return rouge.n(values[0], values[1])
+    // return 0;
+}
+
+const RougeScoreLens: Lens = ({values})=>{
+    const result = calculateRougeScore(values)
+    return (<>Rouge score: {result}</>)
+}
+
+RougeScoreLens.key="RougeScoreView"
+RougeScoreLens.dataTypes=["str"]
+RougeScoreLens.defaultHeight = 48
+RougeScoreLens.minHeight=22
+RougeScoreLens.maxHeight=512
+RougeScoreLens.multi=true
+RougeScoreLens.displayName="ROUGE Score"
+RougeScoreLens.filterAllowedColumns=(allColumns, selectedColumns) => {
```

Comment:
as the rouge score does not make sence to be computed on the same text, it would be good to remove the already selected column from the allowed column.
Should be doable by the `key` attribute.

Also you can simply return an empty array when two columns are selected.